scripts/start/readCheckScenarioConfig.R

```diff
@@ -20,25 +20,25 @@ readCheckScenarioConfig <- function(filename, remindPath = ".", testmode = FALSE
     warning("These titles are too long: ",
             paste0(rownames(scenConf)[nchar(rownames(scenConf)) > 75], collapse = ", "),
             " – GAMS would not tolerate this, and quit working at a point where you least expect it. Stopping now.")
-    errorsfound <- errorsfound + 1
+    errorsfound <- errorsfound + sum(nchar(rownames(scenConf)) > 75)
   }
-  if (any(nchar(rownames(scenConf)) < 4)) {
-    warning("These titles are too short: ",
-            paste0(rownames(scenConf)[nchar(rownames(scenConf)) < 4], collapse = ", "),
-            " – Titles with less than 4 characters may be confused with region names by magclass. Stopping now.")
-    errorsfound <- errorsfound + 1
+  if (any(grepl("^(([A-Z]{3})|(glob))$", rownames(scenConf)))) {
+    warning("These titles may be confused with regions: ",
+            paste0(rownames(scenConf)[grepl("^(([A-Z]{3})|(glob))$", rownames(scenConf))], collapse = ", "),
+            " – Titles with three capital letters or 'glob' may be confused with region names by magclass. Stopping now.")
+    errorsfound <- errorsfound + sum(grepl("^(([A-Z]{3})|(glob))$", rownames(scenConf)))
```

Comment:
I think it would be better to define expressions which are rather complex and used multiple times as variables. So, something like:
```R
tooLong <- nchar(rownames(scenConf)) > 75
if (any(tooLong)) {
…
```
and
```R
regionCodeLike <- grepl("^(([A-Z]{3})|(glob))$", rownames(scenConf))
if (any(regionCodeLike)) {
    …
    errorsfound <- errorsfound + sum(regionCodeLike)
```

if you agree I can also make the changes.